### ISSUE TITLE
Add Socks 5 AND HTTP Proxy Support

### ISSUE THREAD

**Issue by yario-o** [dahquan/rattlesnake#70]
I add socks5 and http support like ogar-feeder-bot(with #HTTP and #SOCKS5)

All is here : 
https://github.com/yario-o/rattlesnake/tree/SocksAndHTTP

**Comment by yario-o** [comment]
now connect function is connect(proxy, mode)

**Comment by yario-o** [comment]
If you don't put #HTTP or #SOCKS5 it will be used for HTTP

**Comment by Wartoz** [comment]
so now we can connect http and socks at same time?

**Comment by yario-o** [comment]
@Wartoz yes

**Comment by yario-o** [comment]
I just add SOCKS4

**Comment by yario-o** [comment]
Now you can do #SOCKS4

**Comment by anonymousxxx02** [comment]
@yario-o i am getting this error

**Comment by yario-o** [comment]
@anonymousxxx02  which error

**Comment by anonymousxxx02** [comment]
ops sorry 
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }

**Comment by anonymousxxx02** [comment]
also this { [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: read ECONNRESET] code: 'ECONNRESET', errno: 'ECONNRESET', syscall: 'read' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: read ECONNRESET] code: 'ECONNRESET', errno: 'ECONNRESET', syscall: 'read' }
{ [Error: socket hang up] code: 'ECONNRESET' }
[Bots Alive: 11]                xxxx died

**Comment by yario-o** [comment]
Do you have bots?

**Comment by yario-o** [comment]
I think it is not working proxy which do this error.
But normally bots work.

**Comment by yario-o** [comment]
You use that : 
https://github.com/yario-o/rattlesnake/tree/SocksAndHTTP
or that : 
https://github.com/yario-o/slither-feeder-bot

**Comment by anonymousxxx02** [comment]
@yario-o yes i have bots but i think they connect only with sock4 and i am using this code https://github.com/l3mpik/slither-feeder-bot

**Comment by easyrider81** [comment]
I get the same error as @anonymousxxx02

**Comment by yario-o** [comment]
It do this error when proxy don't work

**Comment by anonymousxxx02** [comment]
@yario-o I think it doesnt work correct because before the error it say connect with sock4 proxy

**Comment by yario-o** [comment]
I think I find the error I will fix that.

**Comment by anonymousxxx02** [comment]
@yario-o ok

**Comment by yario-o** [comment]
@anonymousxxx02 @easyrider81 I fix the bots
I upload files

**Comment by anonymousxxx02** [comment]
@yario-o still the same thing
`Mode socks4, proxy 109.202.0.167:1080
Mode socks4, proxy 109.202.0.167:1080
Mode socks4, proxy 109.195.131.207:1080
Mode socks4, proxy 109.195.131.207:1080
Mode socks4, proxy 103.242.209.85:1080
Mode socks4, proxy 103.242.209.85:1080
Mode socks4, proxy 103.234.254.99:1080
Mode socks4, proxy 103.234.254.99:1080
Mode socks4, proxy 103.224.28.110:1080
Mode socks4, proxy 103.224.28.110:1080
Mode socks4, proxy 103.195.142.88:9999
Mode socks4, proxy 103.195.142.88:9999
Mode socks4, proxy 12.153.12.125:10200
Mode socks4, proxy 12.153.12.125:10200
Mode socks4, proxy 149.56.172.89:11270
Mode socks4, proxy 149.56.172.89:11270
Mode socks4, proxy 149.56.172.254:11435
Mode socks4, proxy 149.56.172.254:11435
Mode socks4, proxy 149.56.172.167:11348
Mode socks4, proxy 149.56.172.167:11348
Mode socks4, proxy 138.97.236.13:1080
Mode socks4, proxy 138.97.236.13:1080
Mode socks4, proxy 122.200.2.1:1080
Mode socks4, proxy 122.200.2.1:1080
Mode socks4, proxy 153.226.160.37:1080
Mode socks4, proxy 153.226.160.37:1080
Mode socks4, proxy 138.97.236.17:1080
Mode socks4, proxy 138.97.236.17:1080
Mode socks4, proxy 130.211.96.191:3128
Mode socks4, proxy 130.211.96.191:3128
Mode socks4, proxy 14.38.187.171:10200
Mode socks4, proxy 14.38.187.171:10200
Mode socks4, proxy 141.138.156.83:1080
Mode socks4, proxy 141.138.156.83:1080
Mode socks4, proxy 176.109.252.177:1080
Mode socks4, proxy 176.109.252.177:1080
[Error: Rejected (91)]
[Error: Rejected (91)]
Bot die
Bot die
Bot die
{ [Error: tunneling socket could not be established, cause=Parse Error] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: tunneling socket could not be established, cause=read ECONNRESET] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
{ [Error: tunneling socket could not be established, cause=socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
Bot die
Bot die
{ [Error: tunneling socket could not be established, cause=Parse Error] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
Bot die
{ [Error: tunneling socket could not be established, cause=socket hang up] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }
Bot die
{ [Error: tunneling socket could not be established, cause=Parse Error] code: 'ECONNRESET' }
{ [Error: socket hang up] code: 'ECONNRESET' }`

**Comment by yario-o** [comment]
@anonymousxxx02 I do a new update

**Comment by anonymousxxx02** [comment]
@yario-o Ok i am gonna check this later i hope to work this time 😂
